### DISCUSSION TITLE
R2s使用Rclone的copy 复制大文件(>25G)到本地硬盘会死机

### DISCUSSION THREAD

**Discussion by KazeLiu** [messense/aliyundrive-webdav#455] [Q&A]
小文件没什么事情
大文件copy到12G左右就死机了，不知道怎么排查原因
用到的命令类似于 rclone copy -P KazeAliyun:/原地址 /mnt/sda2/Kaze/

**Comment by messense** [comment]
`dmesg` 看下日志是不是 OOM kill 了。

**Reply by KazeLiu** [reply]
难不成没内存了还会把系统杀了？

**Reply by messense** [reply]
自己学习：
https://www.kernel.org/doc/gorman/html/understand/understand016.html
https://www.vpsee.com/2013/10/how-to-configure-the-linux-oom-killer/
